current/wef/collection.json

```diff
@@ -1161,7 +1161,6 @@
     }
   },
   "license": "proprietary",
-  "keywords": [],
```

Comment:
Not sure why but PR also changes a lot of things in the CBCA, CFHP, CFR, floodmaps and wef layers. Could you check why this is the case?